### PR TITLE
Evidently integration

### PR DESCRIPTION
**Pull request by francesco086**
Solves https://github.com/iterative/dvclive/issues/699

### CODE REVIEW

File: content/docs/user-guide/integrations/evidently.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Evidently
+
+DVCLive can be used to track the results
+of[Evidently](https://www.evidentlyai.com/). In the following we demonstrate it
```
Comment (dberenbaum) [inline content/docs/user-guide/integrations/evidently.md:4]:
```suggestion
of [Evidently](https://www.evidentlyai.com/). In the following we demonstrate it
```

File: content/docs/user-guide/integrations/evidently.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+# Evidently
+
+DVCLive can be used to track the results
```
Comment (dberenbaum) [inline content/docs/user-guide/integrations/evidently.md:3]:
```suggestion
[DVCLive](/doc/dvclive) can be used to track the results
```

File: content/docs/user-guide/integrations/evidently.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+There are two ways to track the results of Evidently with DVCLive:
+
+1. you can save the results of each item in the batch in one single experiment
+   (each experiment corresponds to a git commit), in separate steps
+2. or you can save the result of each item in the batch as a separate experiment
+
+We will demonstrate both, and show you how to inspect the results regardless of
+your IDE. However, if you are using VSCode, we recommend using
+[our awesome VSCode extension](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
```
Comment (dberenbaum) [inline content/docs/user-guide/integrations/evidently.md:119]:
```suggestion
the [DVC extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
```

Appreciate the love but technically this is what we are supposed to call it!

**Review by dberenbaum — CHANGES_REQUESTED** [review]
Amazing work @francesco086! 

Would you mind first submitting the notebook in a PR to dvclive in the [examples](https://github.com/iterative/dvclive/tree/main/examples) folder? Then you can link to it in the docs like [this huggingface example](https://github.com/iterative/dvc.org/blob/main/content/docs/dvclive/ml-frameworks/huggingface.md).

Otherwise, I have only a few minor comments if you have time to do one more iteration:
1. To show off the other available interfaces, you could try making tabs like in https://dvc.org/doc/start/experiments/experiment-tracking#tracking.
2. At the top of the notebook, you might want to drop the outputs of the cells or use `pip install -q` so that it's easier to get to the interesting stuff.
3. In the notebook, there are some options to show the outputs more natively. For the single experiment, you can show the plots by including [`notebook=True`](https://dvc.org/doc/dvclive/live#parameters) when initializing DVCLive. For multiple experiments, you could show and manipulate the table in python using [`dvc.api.exp_show()`](https://dvc.org/doc/api-reference/exp_show).

File: content/docs/user-guide/integrations/evidently.md (hunk 1 above)
Comment (francesco086) [reply content/docs/user-guide/integrations/evidently.md]: done

File: content/docs/user-guide/integrations/evidently.md (hunk 2 above)
Comment (francesco086) [reply content/docs/user-guide/integrations/evidently.md]: done

File: content/docs/user-guide/integrations/evidently.md (hunk 3 above)
Comment (francesco086) [reply content/docs/user-guide/integrations/evidently.md]: done

**Comment by francesco086** [comment]
> Amazing work @francesco086!
> 
> Would you mind first submitting the notebook in a PR to dvclive in the [examples](https://github.com/iterative/dvclive/tree/main/examples) folder? Then you can link to it in the docs like [this huggingface example](https://github.com/iterative/dvc.org/blob/main/content/docs/dvclive/ml-frameworks/huggingface.md).
> 
> Otherwise, I have only a few minor comments if you have time to do one more iteration:
> 
> 1. To show off the other available interfaces, you could try making tabs like in https://dvc.org/doc/start/experiments/experiment-tracking#tracking.
> 2. At the top of the notebook, you might want to drop the outputs of the cells or use `pip install -q` so that it's easier to get to the interesting stuff.
> 3. In the notebook, there are some options to show the outputs more natively. For the single experiment, you can show the plots by including [`notebook=True`](https://dvc.org/doc/dvclive/live#parameters) when initializing DVCLive. For multiple experiments, you could show and manipulate the table in python using [`dvc.api.exp_show()`](https://dvc.org/doc/api-reference/exp_show).

Thanks @dberenbaum ! Done all you suggested except for point 1. 
I could do that, but

1. I must confess I find those gif pretty hard to understand, they are too small...
1. if you really insist on that, I need hints on the tools to use to record the video to make an animated gif, I have never done such things before...

For the same reason as above (pictures too large to fit in the webpage), I preferred to leave the native jupyter output in the notebook itself, without putting them in the md. However I added hints on how to do that. Let me know if that is ok, or you prefer otherwise.

**One last thing**: I noticed that colab allows to save a copy of the notebook directly in github. Could you perhaps store it directly in the [example](https://github.com/iterative/dvclive/tree/main/examples) folder, since you probably have the rights to do so?
Once you have done it I can update the colab link.

**Comment by francesco086** [comment]
> No problem, we can go ahead and update in the future if needed. Good feedback on the gif size. If you have other ideas how to show the equivalent workflow in VS Code/Studio, let me know.

I must confess that I have no clue. Perhaps a youtube video would be a better option because it can be enlarged.
However, all of these options are very time consuming and are very hard to maintain (potentially each update could require to re-record the video). What I would do for my own repo is to keep such videos to the bare minimum, as I can't see a way to automate their creation.
 
> Do you mind submitting a PR to the dvclive repo with this notebook saved in the examples folder? That way, you will get the credit for 2 contributions, and it better reflects the real workflow, plus we can review the actual notebook contents if needed.

Didn't think of the notebook review, here you go: https://github.com/iterative/dvclive/pull/723

File: content/docs/user-guide/integrations/evidently.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
+# Evidently
+
+[DVCLive](/doc/dvclive) can be used to track the results of
+[Evidently](https://www.evidentlyai.com/). In the following we demonstrate it
+through an example.
+
+<p align='center'>
+  <a href="https://colab.research.google.com/drive/14usegPOSArF9tdO7NUOPndNk5vuUaYaX?usp=sharing">
```
Comment (dberenbaum) [inline content/docs/user-guide/integrations/evidently.md:8]:
```suggestion
  <a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Evidently.ipynb">
```

**Review by dberenbaum — APPROVED** [review]

**Comment by dberenbaum** [comment]
Thanks again @francesco086, great work!

**Pull request merged**